client/src/components/CurrentWeatherFunctions.ts

```diff
@@ -0,0 +1,32 @@
+type Coordinates = {
+  latitude: number;
+  longitude: number;
+};
+
+export async function getCityCoordinates(city: string | undefined) {
```

Comment:
tady je trosku zvlastni ten typ parametru. Proc je dovolene undefined? To by jsi tu funkci idealne nemel volat vubec. A pokud ano, nemela by urcite pokracovat ve volani toho fetch